### DISCUSSION TITLE
The lost map

### DISCUSSION THREAD

**Discussion by FabioOG** [redeclipse/base#1143] [Maps and Content Showcase]
Some people know it, i work on the lost map since a lot of time, with Trooper and yoho, and other contributions. It's a huge volcanic map where an omega rocket has landed, but the alpha team were already here and cut this place with a magnetic field. The map is made of a lot of small bases mostly made for snipe, all around the map. The dimensions of the map were actually made for my rifle vars, but it works with the default vars too. I tried to most of times put the weapons in a place where it is supposed to be, like a shelf, or a locker, not on the ground. I'll add that I made a lot of secrets in it.
(I thank snipergoth for helping me in the lightning)

![20180521131358](https://user-images.githubusercontent.com/35398459/40430082-8d298bbc-5ea4-11e8-970e-fd0dc7ac4474.png)
![20180521131442](https://user-images.githubusercontent.com/35398459/40430102-a0cb1cbc-5ea4-11e8-9725-14d93840e73f.png)
![20180521131508](https://user-images.githubusercontent.com/35398459/40430120-adcac2be-5ea4-11e8-98fd-919499927b52.png)
![20180521131708](https://user-images.githubusercontent.com/35398459/40430151-bf4856f0-5ea4-11e8-9394-2ebf84f53737.png)
![20180521131752](https://user-images.githubusercontent.com/35398459/40430156-c0959de2-5ea4-11e8-8920-e957fc85bcaa.png)

[lost.zip](https://github.com/red-eclipse/world/files/2031449/lost.zip)

**Comment by SniperGoth** [comment]
Heh, it's the map that i helped with the atmosphere/basic lighting setup 🗡

**Comment by FabioOG** [comment]
Yes, as i said in last line :smiley:

**Comment by FabioOG** [comment]
I corrected some textures and made a version that seem compatible with the 1.9.9 version
[lost2.zip](https://github.com/red-eclipse/world/files/2039284/lost2.zip)

**Comment by ghost** [comment]
It is a wonderful map! There are many secrets and the lightning is very good! =)

**Comment by FabioOG** [comment]
So it is not 1.6 only right ?
I'm waiting for almost 3 weeks that you suppress that label

**Comment by cloudziq** [comment]
map is unfinished, contains many bugs, I  spotted these just by quick playtest (in 5 minutes), lighting is messed, map have 122 light sources and none of them is flagged, many lights are clustered - like 20 lights in 1 square meter, all of this resulting in terrible flickering shadows all over the map.
Other things that I spotted in these 5 minutes:
 -simple, square/blocky geometry
 -strange places with invisible teleporters leading straight to lava
 -small dark spaces/corridors
 -sunlight without visible source
 -lack of map variant and map effects feature
 -invisible wall around map, there should be physical boundary
 -weapons are one on another, making it as lottery
 -very bad occlusion


Map maybe runs in 1.9.9, but its totally unoptimised, it looks more like randomly placed elements on raw map. but don't worry, Im sure you will upgrade it

**Comment by FabioOG** [comment]
What is a light flagged ? :smile: 
Well, i'll try to improve some of the geometry and the lightning.
For information, that tp did'nt lead to lava, it comes from lava, and this lava is clip.
What do you mean by lack of map variant and map effects feature ?
For the weapons, why not ?

**Comment by cloudziq** [comment]
there is some info about lights, variants etc, read carefully ;p 
https://www.redeclipse.net/docs/Information-for-v2-0

...also next time do not post unfinished maps, knowing you and you big mouth which only saying that you can fix it, but you rather make another one in the same quality.. please spare us from this
